### PR TITLE
Add warning for certain generic IDP flows.

### PR DESCRIPTION
**Pull request by yuchenshi**
This shows a warning banner for some flows supported in Auth Emulator only but not in production. We will still let the developer proceed for testing though.

cc'd a few folks for print and UX.

Screen Shot 2020-10-12 at 12 53 18 PM

### CODE REVIEW

**Comment by samhorlbeck** [comment]
Hmm when it's multiple lines, the (i) icon should be aligned with the top. Unfortunately, removing "align-items: center" makes it look bad with a single line.

Maybe put the (i) icon in a div with a height: 100%? That will keep the text center-aligned vertically but should keep the (i) at the top

**Comment by yuchenshi** [comment]
> Hmm when it's multiple lines, the (i) icon should be aligned with the top. Unfortunately, removing "align-items: center" makes it look bad with a single line.
> 
> Maybe put the (i) icon in a div with a height: 100%? That will keep the text center-aligned vertically but should keep the (i) at the top

Updated code and screenshot. I think I've found a much simpler solution.

**Review by mjchristy — APPROVED** [review]
LGTM

**Comment by yuchenshi** [comment]
FYI: Callouts with one line aren't affected.

Screen Shot 2020-10-12 at 1 39 11 PM

File: src/emulator/auth/widget_ui.ts [hunk 1]
```diff
@@ -385,16 +396,21 @@ button {
...
 }
 
+.callout-warning {
+  background: #fff3e0;
+  color: #bf360c;
+}
+
 .callout .content {
   flex: 1;
+  align-self: center;
```
Comment (samhorlbeck) [inline src/emulator/auth/widget_ui.ts:413]: nit: consider alphabetizing these. I started doing this style-- we don't need to continue but worth considering

**Review by samhorlbeck — APPROVED** [review]

**Review by henrypbriffel — APPROVED** [review]

**Review by henrypbriffel — APPROVED** [review]

File: src/emulator/auth/widget_ui.ts (hunk 1 above)
Comment (yuchenshi) [reply src/emulator/auth/widget_ui.ts:413]: Next time I touch this file I'll sort them all

**Pull request merged**